src/v2i-hub/MobilityOperationPlugin/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+PROJECT ( MobilityOperationPlugin VERSION 5.0 LANGUAGES CXX )
```

Comment:
Lets change these to the variable form rather than hard coded in

Reply:
OK this was an oversight. Will fix it